docs/release-info/release-notes.md

```diff
@@ -365,21 +365,6 @@ Check out the examples at [All Druid SQL functions](../querying/sql-functions.md
 
 ## Upgrade notes and incompatible changes
 
-### Upgrade notes
```

Comment:
line 368 needs to be added back

Reply:
@317brian  I am not in favor of adding back a heading that will not have any content in it. Should we change the H2 to just say "Incompatible changes"?

Or put the heading back & mention "No upgrade notes to mention"

Reply:
We can delete it, but there's a reference to it somewhere, which is why the link checker is breaking